### PR TITLE
rtmp-services: add "PhoneLiveStreaming"

### PR DESCRIPTION
**Pull request by davidplappert**
### Description
Add PhoneLiveStreaming to rtmp services list.

### Motivation and Context
Allows live streaming to toll-free telephone systems using OBS Studio.

### How Has This Been Tested?
Local build

### Types of changes
New feature (non-breaking change which adds functionality)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

This PR & Commit preceded PR #5435, #5510 which was closed due to contrib issue. Thank you for your help @derrod with the link to a correct example.

### CODE REVIEW

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -2392,6 +2392,21 @@
                 "max video bitrate": 2500,
                 "max audio bitrate": 128
             }
+        },
+        {
+            "name": "PhoneLiveStreaming",
+            "stream_key_link": "https://app.phonelivestreaming.com/media/rtmp",
+            "servers": [
+                {
+                    "name": "PhoneLiveSreaming",
```
Comment (notr1ch) [inline plugins/rtmp-services/data/services.json:2401]: This appears to be typoed.

**Comment by notr1ch** [comment]
Note that you do not need to keep opening new PRs, you can force push your branch to update the PR. Dropping the "quotes" in the commit title would also be nice.

**Comment by davidplappert** [comment]
> Note that you do not need to keep opening new PRs, you can force push your branch to update the PR. Dropping the "quotes" in the commit title would also be nice.

I'm sorry, I will Google this next time.

**Comment by Fenrirthviti** [comment]
In the future, we could just reopen this one, you didn't need to create a new PR.

**Comment by davidplappert** [comment]
> For the future, we could have just reopened this one, you didn't need to create a new PR.

I apologize.

**Pull request closed without merging**